composer.json

```diff
@@ -60,6 +60,7 @@
     },
     "autoload": {
         "psr-4": {
+            "DoctrineMigrations\\": "migrations/",
```

Comment:
I am not a big fan of adding migrations to plugins

Reply:
I will remove it from autoloading so it will not be available at project using this plugin by default. 

The idea was - project used this plugin before (older version) will have problem with null values at `created_at` during schema update. And copying this migration to project will help.

Reply:
Okay. Appreciate your thoroughness, but I don't think anybody ever used this plugin yet. Also it's not in any stable mode, so it's okay to do these BC breaking things

Reply:
It for us. I will remove then once update plugin at project.